src/client/Microsoft.Identity.Client/Internal/Logger/NullIdentityLogger.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.IdentityModel.Abstractions;
+
+namespace Microsoft.Identity.Client.Internal.Logger
+{
+    internal class NullIdentityLogger : IIdentityLogger
```

Comment:
Shouldn't it be provided in Microsoft.IdentityModel.Logging.Abstractions?

Reply:
Yes, didnt realize it existed.